oojs.js

```diff
@@ -1,12 +1,35 @@
 // Define a Playlist class. It should have...
+class Playlist{
+	constructor(title, songs = [], favoriteSong = 0){
+		this.title = title
+		this.songs = [songs]
```

Comment:
Don't need brackets here. Just make sure that whatever is passed in through `new Playlist( )` is an array.